source/base/Profiler.ooc

```diff
@@ -65,6 +70,8 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
```

Comment:
To bad, that is a common design flaw in many programming languages in my opinion. Maybe call it `freeAll` then. It corresponds to the `resetAll`. In the end I would just like to state that this matter has nothing to do with this PR and should not block it from getting merged. We should never let the optimal solution stop an incremental improvement.